site/docs/v1/tech/tutorials/integrate-laravel-api.md

```diff
@@ -0,0 +1,245 @@
+---
+layout: doc
+title: Integrate Your Laravel API With FusionAuth
+description: Integrate your Laravel API with FusionAuth
+navcategory: getting-started
+prerequisites: Composer, Docker and at least PHP 8.2
+technology: Laravel
+language: PHP
+---
+
+## Integrate Your {{page.technology}} API With FusionAuth
+
+{% include docs/integration/_intro.md %}
```

Comment:
I think you need to include _intro-api.md: https://github.com/FusionAuth/fusionauth-site/blob/master/site/_includes/docs/integration/_intro-api.md